### ISSUE TITLE
Note events don't seem to be handled correctly

### ISSUE THREAD

**Issue by blulere** [KaleidonKep99/OmniMIDI#181]
**Problem**
When scrubbing in PFA, random note events begin to play. I assume they're random as they vary between note on and off events, and sometimes pitch bends.
This error appeared in 13.6 and is still occuring in 13.7.

**Steps to reproduce**
1. Play any normal MIDI in PFA and scrub the timeline
2. Random chords begin to play
3. Scrub harder
4. The whole song pitches down a semitone

**Expected behavior**
The MIDI should sound like this (recorded in 13.5).
https://user-images.githubusercontent.com/40177441/109173414-b79af280-777b-11eb-9de3-d15cd31aeb59.mp4

**Screenshots/Videos**
Video 1 contains a quick screengrab I was going to submit to Discord
https://user-images.githubusercontent.com/40177441/109173812-2b3cff80-777c-11eb-8b59-00dde53570b5.mp4
Video 2 contains the pitch shifting
https://streamable.com/vt0keq
**Environment**
 - Operating system: Windows 8.1 Build 9600
 - CPU: Intel Core i5-2520M
 - RAM: 4GB DDR3 RAM
 - MIDI application: PFA 1.1.0
 - Version of the driver: 13.6 - 13.7

**Additional context**
Reset of settings did not help. x86 driver still unregistering. All videos were recorded with DirectX Audio, but issues still remained when setting all settings to default.
I have also experienced stuttering in 13.5-13.7 when the polyphony > about 50.

**Comment by KaleidonKep99** [comment]
It doesn't seem to occur when the player is patched with the WinMMWRP, so it might be an issue in the modMessage function used by Windows Multimedia.

I'll look into it.

**Comment by KaleidonKep99** [comment]
Let me know if it works now.
I tested it myself and I don't have the issue anymore, I should've tested before releasing 13.7 though! 😂

**Comment by blulere** [comment]
https://streamable.com/856fuu
This issue seems to solely affect PFA. If there was a debugging option that allowed me to view the inputs OmniMIDI saw, I think I might be able to close the issue down...

**Comment by KaleidonKep99** [comment]
> 
> 
> https://streamable.com/856fuu
> This issue seems to solely affect PFA. If there was a debugging option that allowed me to view the inputs OmniMIDI saw, I think I might be able to close the issue down...

Is this on OmniMIDI 13.8?

**Comment by blulere** [comment]
Yep, sorry for the late reply.

After doing some further experimenting, I found out that past note events appear to be playing on new note on events. Somehow I managed to find this out in Domino, so I guess my previous claim was wrong. 
Enabling "Only release the oldest instance of a note upon note-off event" made things worse, as it appeared that note-off events were not being handled correctly.
Sadly I couldn't get a video of the effect occuring in Domino as it seemed to only happen once, but the steps I took to get it happen were to:
1. Open Domino.
2. Load OmniMIDI with GSm
3. Load a new project
4. Go into the Program Change on the first track 
![image](https://user-images.githubusercontent.com/40177441/109679752-c74b7a00-7b73-11eb-84b1-5c79849c9ac9.png)
5. Mess around with the instruments in the Chromatic Perc section and eventually it may happen.
I was curious if this was a one-MIDI thing, so I played the FLP version of Pi:
https://streamable.com/epuiml
